README.md

```diff
@@ -53,6 +53,10 @@ Start with [**'Hello, Client!'**](https://developer.paypal.com/braintree/docs/st
 
 Next, read the [**full documentation**](https://developer.paypal.com/braintree/docs/guides/overview) for information about integration options, such as Drop-In UI, PayPal and credit card tokenization.
 
+## Upgrade Your SDK Version
+
+See [**Android Developer Guidelines**](https://developer.android.com/build/dependencies) for details on how to add and manage build dependencies. 
```

Comment:
Should we detail the steps on how to upgrade versions of the SDK? I'm not sure if there's helpful information on the dependencies documentation.

We could call out: incrementing the SDK version number, checking the changelog, following the migration guide for major version upgrades.

Reply:
@tdchow for sure, that's a great callout - i've added some more detail in 8950a254d4e5e8b56d293e76a9c0432772cc0da9. is there anything else you think i can mention 👀